CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## [Unreleased]
 
+## [2.0.0-pre-rc.21.1] - 2024-05-01
+
+- add RawGenesisBlock to schema (#4538)
+- remove nested option on `TransactionEventFilter::block_height` (#4538)
+
 ## [2.0.0-pre-rc.21] - 2024-04-19
```

Comment:
Should we initially create `2.0.0-per-rc.21.1`? 
This way `2.0.0-pre-rc.21` would mean most recent `rc.21` version.

Reply:
yes, I think we should. But I didn't anticipate this